### DISCUSSION TITLE
starknet.py after cairo-lang is deprecated

### DISCUSSION THREAD

**Discussion by ondrejsojka** [software-mansion/starknet.py#1099] [General]
Starknet Dev News from 29.6.:
> We’d like to inform the community that Starknet v0.12.0 is the last version to be officially supported on cairo-lang; starting from v0.12.1, we may not maintain cairo-lang to support new Starknet versions.

Will starknet.py be rewritten to use starknet-in-rust or will we have to rewrite our scripts to Rust?

**Comment by drknzz** [comment]
Hey, we've managed to cut-off from **cairo-lang** and not use it for some time now. It's currently only used as a dev dependency, to compile contracts for testing.
So to answer, Starknet.py will stay the way it is :)

**Comment by ondrejsojka** [comment]
That's super nice to hear, thanks!

st 5. 7. 2023 v 11:17 odesílatel Kamil Jankowski ***@***.***>
napsal:
